crates/generated_parser/src/early_error_checker.rs

```diff
@@ -0,0 +1,646 @@
+use crate::context_stack::{
+    BindingInfo, BindingKind, BindingsIndex, BreakOrContinueIndex, ContextMetadata, LabelIndex,
+    LabelInfo, LabelKind,
+};
+use crate::declaration_kind::DeclarationKind;
+use crate::early_errors::*;
+use crate::error::{ParseError, Result};
+use crate::Token;
+use ast::arena;
+use ast::source_atom_set::{SourceAtomSet, SourceAtomSetIndex};
+use std::cell::RefCell;
+use std::rc::Rc;
+
+// Internal Helper Methods for Early Error Checker Labels/Controls
+
+/// Internal Label/Control Helper method:
```

Comment:
If you move those methods out of `trait`, I think this comment is unnecessary,
because lack of `pub` already means the same thing.